worker/src/main/java/gov/cms/ab2d/worker/processor/ProgressTracker.java

```diff
@@ -73,7 +63,7 @@ public void addPatients(List<CoverageSummary> coverage) {
      * @return number of patients
      */
     public int getTotalCount() {
-        return patients.size();
+        return metadataProcessedCount;
```

Comment:
Why not expected beneficiaries?

Reply:
Ok, wasn't quite sure the best replacement.  Fixed.